src/sider/frisktilarbeid/VedtakDatoer.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import dayjs from "dayjs";
+import { DatePicker, useDatepicker } from "@navikt/ds-react";
+import { useController } from "react-hook-form";
+import { FattVedtakSkjemaValues } from "./FattVedtak";
+
+const texts = {
+  fraDatoMissing: "Vennligst angi dato",
+  fraDatoLabel: "Friskmeldingen gjelder fra",
+  fraDatoDescription: "Dette er datoen vedtaket starter",
+  tilDatoLabel: "Til dato (automatisk justert 12 uker frem)",
+  tilDatoDescription: "Dette er datoen vedtaket slutter",
+};
+
+interface VedtakDatoerProps {
+  tilDato: Date | undefined;
+  onFraDatoChanged: (fom: Date) => void;
+}
+
+export const VedtakDatoer = ({
+  tilDato,
+  onFraDatoChanged,
+}: VedtakDatoerProps) => {
+  const { field, fieldState } = useController<FattVedtakSkjemaValues>({
+    name: "fraDato",
+    rules: {
+      required: texts.fraDatoMissing,
+    },
+  });
+  const fraDatoDatePicker = useDatepicker({
+    onDateChange: (date: Date | undefined) => {
+      if (date) {
+        field.onChange(dayjs(date).format("YYYY-MM-DD"));
+        onFraDatoChanged(date);
+      }
+    },
+  });
+  const tilDatoDatePicker = useDatepicker();
+
+  return (
+    <>
+      <DatePicker {...fraDatoDatePicker.datepickerProps} strategy="fixed">
+        <DatePicker.Input
+          {...fraDatoDatePicker.inputProps}
+          label={texts.fraDatoLabel}
+          description={texts.fraDatoDescription}
+          error={fieldState.error?.message}
+        />
```

Comment:
Skal vi gjøre det umulig å velge dato tilbake i tid, eller umulig å velge helger?

Reply:
Ja, vi bær sperre for tilbake i tid, kan fikse det. Tror vi eventuelt får se på rød-dager problematikk senere :sweat_smile: